Sources/Testing/Testing.docc/MigratingFromXCTest.md

```diff
@@ -27,6 +27,27 @@ Before the testing library can be used, it must be added as a dependency of your
 Swift package or Xcode project. For more information on how to add it, see the
 [Getting Started](doc:TemporaryGettingStarted) guide.
 
+### Importing the testing library
+
+XCTest and the testing library are available from different modules. Instead of importing the XCTest module, import the Testing module:
+
+@Row {
+  @Column {
+    ```swift
+    // Before
+    import XCTest
+    ```
+  }
+  @Column {
+    ```swift
+    // After
+    import Testing
+    ```
+  }
+}
+
+- Note: You can mix usage of XCTest and this testing library in the same file if you want.
```

Comment:
I don't think this needs to be a note: it causes the text to be significantly emphasized in a grey box and I don't think that's needed here.

It's important to speak with a consistent voice in the documentation. Remember, we avoid "you" here unless we're giving the developer specific instructions. How about:

> A single source file can contain tests written with XCTest as well as other tests written with the testing library. Import both XCTest and Testing if a source file contains mixed test content.

Reply:
Great, fixed!